src/payment/strategies/digitalriver/digitalriver-payment-strategy.ts

```diff
@@ -175,4 +178,53 @@ export default class DigitalRiverPaymentStrategy implements PaymentStrategy {
 
         return this._digitalRiverInitializeOptions;
     }
+
+    private _loadWidget(options: PaymentInitializeOptions): Promise<void> {
+        return this._store.dispatch(this._paymentMethodActionCreator.loadPaymentMethod(options.methodId))
+            .then(async state => {
+                const {clientToken} = state.paymentMethods.getPaymentMethodOrThrow(options.methodId);
+
+                if (!clientToken) {
+                    throw new MissingDataError(MissingDataErrorType.MissingPaymentMethod);
+                }
+
+                const billing = state.billingAddress.getBillingAddressOrThrow();
+                const customer = state.customer.getCustomerOrThrow();
+                const {paymentMethodConfiguration} = this._getDigitalRiverInitializeOptions().configuration;
+                this._digitalRiverCheckoutId = JSON.parse(clientToken).checkoutId;
+                this._digitalRiverSessionId = JSON.parse(clientToken).sessionId;
+                this._submitFormEvent = this._getDigitalRiverInitializeOptions().onSubmitForm;
+                const configuration = {
+                    sessionId: JSON.parse(clientToken).sessionId,
+                    options: {...this._getDigitalRiverInitializeOptions().configuration},
+                    billingAddress: {
+                        firstName: billing.firstName,
+                        lastName: billing.lastName,
+                        email: billing.email || customer.email,
+                        phoneNumber: billing.phone,
+                        address: {
+                            line1: billing.address1,
+                            line2: billing.address2,
+                            city: billing.city,
+                            state: billing.stateOrProvinceCode,
+                            postalCode: billing.postalCode,
+                            country: billing.countryCode,
+                        },
+                    },
+                    paymentMethodConfiguration,
+                    onSuccess: (data?: OnSuccessResponse) => {
+                        this._onSuccessResponse(data);
+                    },
+                    onReady: (data?: OnReadyResponse) => {
+                        this._onReadyResponse(data);
+                    },
+                    onError: (error: OnCancelOrErrorResponse) => {
+                        this._getDigitalRiverInitializeOptions().onError?.(new Error(this._getErrorMessage(error)));
+                    },
+                };
+                this._digitalRiverDropComponent = await this._getDigitalRiverJs().createDropin(configuration);
+
+                return Promise.resolve(this._digitalRiverDropComponent.mount(this._getDigitalRiverInitializeOptions().containerId));
```

Comment:
Minor, but it's good to keep things readable. Ideally, lines shouldn't exceed 120 chars. 

```js
const { containerId } = this._getDigitalRiverInitializeOptions()
return Promise.resolve(this._digitalRiverDropComponent.mount(containerId));